apps/test/unit/turtle/turtleTest.js

```diff
@@ -175,6 +175,32 @@ describe('Artist', () => {
     });
   });
 
+  describe('pointTo', () => {
+    it('can point to 45 degrees', () => {
+      let angle = 45;
+      let blockId = "block_id_4";
+      let artist = new Artist();
+
+      artist.visualization = new Artist.Visualization();
+      const pointToSpy = sinon.spy(artist.visualization, 'pointTo');
+      artist.step('PT', [angle, blockId]);
+
+      expect(pointToSpy).to.be.have.been.calledWith(angle);
+    });
+
+    it('can point to a specific direction', () => {
+      const angle = 45;
+      const blockId = "block_id_8";
+      let artist = new Artist();
+
+      artist.visualization = new Artist.Visualization();
+      artist.visualization.angle = 45;
```

Comment:
This should be something other than 45, to ensure that the `PT` step actually changes the angle.

Reply:
Modified test to include different angles.